nx-X11/programs/Xserver/hw/nxagent/NXshm.c

```diff
@@ -1339,161 +489,3 @@ ProcShmDispatch (client)
     }
 }
 
-static void
-SShmCompletionEvent(from, to)
-    xShmCompletionEvent *from, *to;
-{
-    to->type = from->type;
-    cpswaps(from->sequenceNumber, to->sequenceNumber);
-    cpswapl(from->drawable, to->drawable);
-    cpswaps(from->minorEvent, to->minorEvent);
-    to->majorEvent = from->majorEvent;
-    cpswapl(from->shmseg, to->shmseg);
-    cpswapl(from->offset, to->offset);
-}
-
-static int
-SProcShmQueryVersion(client)
-    register ClientPtr	client;
-{
-    register int n;
-    REQUEST(xShmQueryVersionReq);
-
-    swaps(&stuff->length, n);
-    return ProcShmQueryVersion(client);
-}
-
-static int
-SProcShmAttach(client)
-    ClientPtr client;
-{
-    register int n;
-    REQUEST(xShmAttachReq);
-    swaps(&stuff->length, n);
-    REQUEST_SIZE_MATCH(xShmAttachReq);
-    swapl(&stuff->shmseg, n);
-    swapl(&stuff->shmid, n);
-    return ProcShmAttach(client);
-}
-
-static int
-SProcShmDetach(client)
-    ClientPtr client;
-{
-    register int n;
-    REQUEST(xShmDetachReq);
-    swaps(&stuff->length, n);
-    REQUEST_SIZE_MATCH(xShmDetachReq);
-    swapl(&stuff->shmseg, n);
-    return ProcShmDetach(client);
-}
-
-static int
-SProcShmPutImage(client)
-    ClientPtr client;
-{
-    register int n;
-    REQUEST(xShmPutImageReq);
-    swaps(&stuff->length, n);
-    REQUEST_SIZE_MATCH(xShmPutImageReq);
-    swapl(&stuff->drawable, n);
-    swapl(&stuff->gc, n);
-    swaps(&stuff->totalWidth, n);
-    swaps(&stuff->totalHeight, n);
-    swaps(&stuff->srcX, n);
-    swaps(&stuff->srcY, n);
-    swaps(&stuff->srcWidth, n);
-    swaps(&stuff->srcHeight, n);
-    swaps(&stuff->dstX, n);
-    swaps(&stuff->dstY, n);
-    swapl(&stuff->shmseg, n);
-    swapl(&stuff->offset, n);
-    return ProcShmPutImage(client);
-}
-
-static int
-SProcShmGetImage(client)
-    ClientPtr client;
-{
-    register int n;
-    REQUEST(xShmGetImageReq);
-    swaps(&stuff->length, n);
-    REQUEST_SIZE_MATCH(xShmGetImageReq);
-    swapl(&stuff->drawable, n);
-    swaps(&stuff->x, n);
-    swaps(&stuff->y, n);
-    swaps(&stuff->width, n);
-    swaps(&stuff->height, n);
-    swapl(&stuff->planeMask, n);
-    swapl(&stuff->shmseg, n);
-    swapl(&stuff->offset, n);
-    return ProcShmGetImage(client);
-}
-
-static int
-SProcShmCreatePixmap(client)
-    ClientPtr client;
-{
-    register int n;
-    REQUEST(xShmCreatePixmapReq);
-    swaps(&stuff->length, n);
-    REQUEST_SIZE_MATCH(xShmCreatePixmapReq);
-    swapl(&stuff->pid, n);
-    swapl(&stuff->drawable, n);
-    swaps(&stuff->width, n);
-    swaps(&stuff->height, n);
-    swapl(&stuff->shmseg, n);
-    swapl(&stuff->offset, n);
-    return ProcShmCreatePixmap(client);
-}
-
-static int
-SProcShmDispatch (client)
```

Comment:
SProcShmDispatch in NXshm.c:

> Modified, but not kept?

Another good catch!!! You are awesome! Fixed.